docs/network_firewall.rst

```diff
@@ -28,14 +28,7 @@ If you are new to pfSense or firewall management in general, we
 recommend the following resources:
```

Comment:
resources->resource

singular now

Reply:
(fixed in 3cf2ffeb29b033792b07f1cd5ddaa125b28b448a)

Reply:
Thanks!